resources/views/layout/master.blade.php

```diff
@@ -12,20 +13,69 @@
     <link
         href="https://fonts.googleapis.com/css?family=Nunito:200,200i,300,300i,400,400i,600,600i,700,700i,800,800i,900,900i"
         rel="stylesheet">
-        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css" integrity="sha384-xOolHFLEh07PJGoPkLv1IbcEPTNtaed2xpHsD9ESMhqIYd0nLMwNLD69Npy4HI+N" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css"
+        integrity="sha384-xOolHFLEh07PJGoPkLv1IbcEPTNtaed2xpHsD9ESMhqIYd0nLMwNLD69Npy4HI+N" crossorigin="anonymous">
     <!-- Custom styles for this template -->
     <link href="{{ asset('css/sb-admin-2.min.css') }}" rel="stylesheet">
 </head>
+
 <body id="page-top">
 
-        <div class="card-header py-3">
-            <h6 class="m-0 font-weight-bold text-primary my-3">Bulletin Board</h6>
+    <div class="card-header py-3 d-flex justify-content-between my-3 align-items-center">
+        <h4 class="m-0 font-weight-bold text-primary ">
+            <a href="{{route('post#postList')}}" class=" text-decoration-none">Bulletin Board</a>
+        </h4>
+        <div class="dropdown dropleft">
+            <a class="" type="button" data-toggle="dropdown" aria-expanded="false">
+                @auth
+                    @if (auth()->user()->img)
+                        <img src="{{ asset('storage/images/' . auth()->user()->img) }}" height="50px" alt="">
+                    @else
+                        <img src="{{ asset('img/default.png') }}" height="50px" alt="">
+                    @endif
+                @else
+                    <img src="{{ asset('img/default.png') }}" height="50px" alt="">
+                @endauth
+
+            </a>
+            <div class="dropdown-menu">
+                @auth
+                    <button class="dropdown-item" type="button">{{ auth()->user()->name }}</button>
+
+                    <form method="GET" action="{{ route('user#detail', auth()->user()->id) }}">
+                        @csrf
```

Comment:
Just add url in button  or use <a href="route(routeName)"> 
No need to add form and @csrf for get method.